src/components/Button/Button.react.js

```diff
@@ -22,6 +22,7 @@ type PropsForAll = {|
   +isDropdownToggle?: boolean,
   +to?: string,
   +isOption?: boolean,
+  +rootRef?: Function,
```

Comment:
Let's type this a little more strictly if possible i.e. `(a: string, b: bool) => void`